site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -215,7 +215,6 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.updateStrategy`                        | The [strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) to apply to the ping deployment                                                                                          | `{}`                               |
 | `agones.ping.updateStrategy`                             | The [strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) to apply to the allocator deployment                                                                                     | `{}`                               |
 
-{{% feature publishVersion="1.28.0" %}}
```

Comment:
This one is a bit of a special case (sorry, not covered in the release notes).

For this block, take the config options in the table below, and integrate them into the table above, then increment the `publishVersion` by one minor version (so 1.29.0). We always just keep a block for new config options, rather than replicate the entire massive table.

Reply:
Done